popSimManu.tex

```diff
@@ -557,10 +583,11 @@ \section*{Discussion}
 
 Here we have described the first major product from the PopSim Consortium:
 the \stdpopsim library. We have founded the Consortium with a number of specific goals in mind:
-standardization of simulation within the computational genetics community,
+standardization of simulation within the population genetics community,
```

Comment:
oh good catch - we are totally not working on alignment or protein structure prediction